src/whatsnew.adoc

```diff
@@ -366,208 +367,6 @@ been established.
 
 <<<
 
-[appendix]
```

Comment:
🗺️ An analogous table is now available in the binary encoding section, so I've removed this one rather than update it.